samba/DOCS.md

```diff
@@ -25,6 +25,7 @@ This addon exposes the following directories over smb (samba):
 Directory | Description
 -- | --
 `addons` | This is for your local add-ons.
+`addon_configs` | This is for your the configuration files or your add-ons.
```

Comment:
```suggestion
`addon_configs` | This is for your add-on's configuration files.
```

Reply:
That implies singular, which it isn't?

Reply:
Right, I think this should do it.

```suggestion
`addon_configs` | This is for your add-ons' configuration files.
```

Reply:
Or I guess the of form which might be nicer? Either way works for me.

```suggestion
`addon_configs` | This is for the configuration files of your add-ons.
```